CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
+- to designate important Creator Components, Objets, etc. Example: __Scene__, __Sprite Component__
```

Comment:
Component name should be one world, respect the form when they are used in code

`__SpriteComponent__`

Reply:
Is it better that __Sprite__ component? For example:

Click the __Add Component__ button at the bottom of the __Properties__ panel and select __ScrollBar__ from __UI Component__. You can then add the __ScrollBar__ component to the node.

Reply:
The problem with __SpriteComponent__ is that even though the readers of these docs are developers, that it's confusing to the eyes 2 capital letters without a space. At least when reading on a screen, I think.

Reply:
> it's confusing to the eyes 2 capital letters without a space. At least when reading on a screen, I think.

For developers, I don't think it's a problem, by the way, in Creator 2.x, the class name is `Sprite`, in Creator 3d, the class name is currently `SpriteComponent`, I'm thinking of keeping the same name as in 2.x.

I think the rule should be :
1. Use code wrapper  \` for code related names, like class name, property name and function name.
2. Use bold for important concept or names in the editor UI.

E.g.

- `SpriteComponent`
- `onEnable`
- `SpriteComponent.spriteFrame`
- __Sprite inspector__ or __Sprite__ inspector
- __Asset panel__ or __Asset__ panel